README.md

```diff
@@ -1,47 +1,164 @@
 # Module
-[![GitHub tag](https://img.shields.io/github/tag/qbeyond/terraform-module-template.svg)](https://registry.terraform.io/modules/qbeyond/terraform-module-template/provider/latest)
-[![License](https://img.shields.io/github/license/qbeyond/terraform-module-template.svg)](https://github.com/qbeyond/terraform-module-template/blob/main/LICENSE)
+[![GitHub tag](https://img.shields.io/github/tag/qbeyond/terraform-azurerm-entra-domain-services.svg)](https://registry.terraform.io/modules/qbeyond/entra-domain-services/azurerm/latest)
+[![License](https://img.shields.io/github/license/qbeyond/terraform-azurerm-entra-domain-services.svg)](https://github.com/qbeyond/terraform-azurerm-entra-domain-services/blob/main/LICENSE)
```

Comment:
Watch out for the correct links. In this case I think you swapped azurerm and azapi in the module name